testing/src/cypress/e2e/02-institution/04-manage-location.cy.ts

```diff
@@ -8,20 +8,30 @@ import data from "../data/institution/manage-location.json";
 import Authorization, {
   ClientId,
 } from "../../custom-command/common/authorization";
+import ManageInstitutionObject from "../../page-objects/Institution-objects/ManageInstitutionObject";
 
 const dashboardInstitutionObject = new DashboardInstitutionObject();
 const institutionManageLocationObject = new ManageLocationObject();
 const institutionHelperActions = new InstitutionHelperActions();
+const manageInstitutionObject = new ManageInstitutionObject();
+
 function loginAndClickOnManageInstitution() {
   institutionHelperActions.loginIntoInstitutionSingleLocation();
   dashboardInstitutionObject.manageInstitutionButton().click();
 }
 
-function loginAndClickOnEditLocation() {
+const enum SideBarMenuItems {
```

Comment:
This enum seems to be something to be shared, for instance, as a parameter for the method `clickOnSideBar`.

Reply:
Updated to be on the shared method